src/migrations/20180821140300_markets_openInterest.ts

```diff
@@ -1,13 +1,13 @@
 import * as Knex from "knex";
 import { promisify } from "util";
-import { updateOpenInterest } from "../blockchain/log-processors/order-filled/update-volumetrics";
+import { updateMarketOpenInterest } from "../blockchain/log-processors/order-filled/update-volumetrics";
 
 exports.up = async (knex: Knex): Promise<any> => {
   return knex.schema.hasColumn("markets", "openInterest").then(async(exists) => {
     if (!exists) await knex.schema.table("markets", (t) => t.string("openInterest").defaultTo("0"));
     const markets = await knex.select("marketId").from("markets");
     for (const market of markets) {
-      await promisify(updateOpenInterest)(knex, market.marketId);
+      await promisify(updateMarketOpenInterest)(knex, market.marketId);
```

Comment:
promisify on a non-callback function? (this PR doesn't introduce this error)

Reply:
Fixed